src/data_generators/analysis_example_data_design.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data design for a quick eval to establish analysis practice."""
+
+from pyDOE import lhs
```

Comment:
Is this module used anywhere?

Reply:
Thanks a lot Rieman for the careful review!  Removed this line.